app/views/needs/revisions/_new_note_form.html.erb

```diff
@@ -1,11 +1,16 @@
-<%= semantic_form_for(:note, url: notes_url) do |f| %>
-  <%= f.inputs do %>
+<%= form_for(:note, url: notes_url) do |f| %>
+  <fieldset>
     <%= hidden_field_tag :content_id, need.content_id %>
-    <%= f.input :text, :label => 'Note', :as => :text, :hint => "Don't forget to save your note when you have finished.", input_html: { class: "col-md-12 add-bottom-margin" } %>
-  <% end %>
-  <%= f.action :submit,
-               :button_html => {
-                 :value => "Add note",
-                 :class => "btn btn-primary"
-               } %>
+    <div class="form-group">
+      <span class="form-label">
+        <%= f.label :text, "Note" %>
+      </span>
+      <span class="form-wrapper">
+        <%= f.text_area :text, rows: 20, class: "col-md-12 add-bottom-margin" %>
```

Comment:
Might be missing something but can't see where the `rows: 20` came from?

Reply:
That's because in formtastic it was a large text area, I think formtastic had some sort of implicit formatting on it to make it big. So I added that to match it.